src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequence.test.tsx

```diff
@@ -14,30 +14,74 @@
  * limitations under the License.
  */
 
-import React from 'react';
+import React, { createContext } from 'react';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
 import { setTimeout } from 'timers/promises';
 import { render, fireEvent, createEvent, act } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { faker } from '@faker-js/faker';
 import random from 'lodash/random';
 
-import BlastInputSequence from './BlastInputSequence';
+import BlastInputSequence, {
+  type Props as BlastInputSequenceProps
+} from './BlastInputSequence';
+
+import { initialState as initialBlastFormState } from 'src/content/app/tools/blast/state/blast-form/blastFormSlice';
+
+import mockBlastSettingsConfig from 'tests/fixtures/blast/blastSettingsConfig.json';
 
 const testInput = 'AGCT'; // it shouldn't even matter for testing purposes
 
+jest.mock('src/content/app/tools/blast/state/blast-api/blastApiSlice', () => {
+  return {
+    useBlastConfigQuery: () => ({ data: mockBlastSettingsConfig })
+  };
+});
+
 const commonProps = {
   onCommitted: jest.fn(),
   sequenceType: 'dna' as const
 };
 
+const mockState = {
+  blast: {
+    blastForm: initialBlastFormState
+  }
+};
+
+const emptyTestContext = {
+  sequenceValidityStatus: undefined,
+  updateSequenceValidity: jest.fn()
+};
+let testContext: typeof emptyTestContext;
```

Comment:
We don't need to reassign `emptyTestContext` to another variable. Instead use it directly?